apps/testnorge-skd/skd-core/src/main/java/no/nav/registre/skd/service/EksisterendeIdenterService.java

```diff
@@ -298,6 +287,18 @@ void behandleFarskapMedmorskap(
         oppdaterMeldingerMedFarsFnr(meldinger, barnMedFedre, antallMeldinger, brukteIdenterIDenneBolken);
     }
 
+    private String hentMorFnrRelasjon(RelasjonsResponse relasjonerTilBarn, String morFnr) {
+        for (var relasjon : relasjonerTilBarn.getRelasjoner()) {
+            if (RELASJON_FAR.equals(relasjon.getTypeRelasjon())) {
+                break;
```

Comment:
`break;` vil ikke være nok her. I original kode så hoppet det videre til neste ident i foedteIdenter (iterateFoedteIdenter) og koden `var farFnr = foedselService.findFar(morFnr, foedtIdent, levendeIdenterINorge, new ArrayList<>());`... ble ikke kjørt. Hvis du bare bruker break her så vil antagelig `morFnr` bli returnert med en null verdi. Deretter vil koden `var farFnr = foedselService.findFar(morFnr, foedtIdent, levendeIdenterINorge, new ArrayList<>());`... bli kjørt, noe som ikke skal skje.

Du kunne kanskje flyttet `var morFnr = "";` til øverst i `hentMorFnrRelasjon` og returnert null i stedet for `break;`. Du kunne så sjekket resultatet fra metoden og hvis det er lik null så kjører du `continue iterateFoedteIdenter;`:
```
morFnr = hentMorFnrRelasjon(relasjonerTilBarn, morFnr);
if(morFnr == null){
continue iterateFoedteIdenter;
}
```

Reply:
Åja skjønner! Høres bra ut, skal fikse! :D Isåfall vil en vanlig `continue;` fungere, siden det bare er den ene for loopen i metoden ikkesant? Sonar likte ikke den navngitte løkka forrige gang jeg holdt på der 😅

Reply:
Ja, en vanlig `continue` bør funke 😄